### PR TITLE
Add inParens, inSquareParens and inBlock

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: library/src/scala/tasty/util/ShowSourceCode.scala [hunk 1]
```diff
@@ -35,6 +35,27 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
...
+      this += "("
+      body
+      this += ")"
+    }
+
+    def inSquareParens(body: => Unit): Buffer = {
+      this += "["
+      body
+      this += "]"
+    }
```
Comment (liufengyun) [inline library/src/scala/tasty/util/ShowSourceCode.scala]: `inBrackets` to match `inParens`?

File: library/src/scala/tasty/util/ShowSourceCode.scala [hunk 2]
```diff
@@ -35,6 +35,27 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
...
+
+    def inBlock(body: => Unit): Buffer = {
+      this += " {"
+      indented {
+        this += lineBreak()
+        body
+      }
+      this += lineBreak() += "}"
+    }
+
```
Comment (liufengyun) [inline library/src/scala/tasty/util/ShowSourceCode.scala]: `inBraces` to match `inParens`?

**Comment by nicolasstucki** [comment]
Rebased

File: library/src/scala/tasty/util/ShowSourceCode.scala (hunk 1 above)
Comment (nicolasstucki) [reply library/src/scala/tasty/util/ShowSourceCode.scala]: Will change in next pr

**Pull request merged**